Character.cpp

```diff
@@ -97,21 +188,12 @@ void Character::attackInternal(Character& other)
             a) your stats are restored to their initial value if they are lower than it.
             b) your stats are boosted 10%
             c) the initial value of your stats is updated to reflect this boosted stat for the next time you defeat another character.
-      */
-        assert(false);
+        */
+
+        levelUpHitPoints();
+        levelUpArmor();
+        levelUpAttackDamage();
```

Comment:
All of these functions do the SAME thing, just with different variable names.
use a SINGLE free function to do the same thing.  pass your params by reference.
- [ ] revise.

Reply:
combined into one levelUp()